test/build-tasks-sample/.config/dotnet-tools.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "neo.express": {
+      "version": "3.5.19",
```

Comment:
can you update to new versions? Same for one below.

Reply:
OK